src/integration-test/resources/db/mod/reports/UnconfirmedAttendanceReportITest_typical.sql

```diff
@@ -0,0 +1,96 @@
+       ('041500018', 'CName18', 'CSurname18', '1980-01-01', 'addressLine1', 'CH1 2AN', true),
+       ('041500019', 'CName19', 'CSurname19', '1980-01-01', 'addressLine1', 'CH1 2AN', true),
+       ('041500020', 'CName20', 'CSurname20', '1980-01-01', 'addressLine1', 'CH1 2AN', true);
+
+
+INSERT INTO juror_mod.juror_pool (owner, juror_number, pool_number, status, is_active, next_date)
+VALUES ('415', '041500001', '415240101', 2, true, current_date),
+       ('415', '041500002', '415240101', 2, true, current_date),
+       ('415', '041500003', '415240101', 2, true, current_date),
+       ('415', '041500004', '415240101', 2, true, current_date),
+       ('415', '041500005', '415240101', 2, true, current_date),
+       ('415', '041500006', '415240101', 2, true, current_date),
+       ('415', '041500007', '415240101', 2, true, current_date),
+       ('415', '041500008', '415240101', 2, true, current_date),
+       ('415', '041500009', '415240101', 2, true, current_date),
+       ('415', '041500010', '415240101', 2, true, current_date),
+       ('415', '041500011', '415240102', 2, true, current_date),
+       ('415', '041500012', '415240102', 2, true, current_date),
+       ('415', '041500013', '415240102', 2, true, current_date),
+       ('415', '041500014', '415240102', 2, true, current_date),
+       ('415', '041500015', '415240102', 2, true, current_date),
+       ('415', '041500016', '415240102', 2, true, current_date),
+       ('415', '041500017', '415240103', 2, true, current_date),
+       ('415', '041500018', '415240103', 2, true, current_date),
+       ('415', '041500019', '415240103', 2, true, current_date),
+       ('415', '041500020', '415240103', 2, true, current_date);
+
+
+INSERT INTO juror_mod.judge (id, owner, code, description)
+VALUES (1, '415', 'JUD1', 'The Judge');
+INSERT INTO juror_mod.courtroom (id, loc_code, room_number, description)
+VALUES (1, '415', '1', 'The Courtroom');
+INSERT INTO juror_mod.trial (trial_number, loc_code, trial_start_date, trial_type, description, courtroom, judge)
+VALUES ('111111', '415', current_date, 'CRI', 'The Trial', 1, 1),
+       ('111112', '415', current_date, 'CRI', 'The Trial 2', 1, 1),
+       ('111113', '415', current_date, 'CIV', 'The Trial 3', 1, 1);
+
+
+INSERT INTO juror_mod.appearance (attendance_date, juror_number, pool_number, loc_code, time_in, time_out, trial_number,
+                                  appearance_stage)
+VALUES (current_date - 1, '041500001', '415240101', '415', '09:30:00', '17:30:00', '111111', 'CHECKED_OUT'),
+       (current_date - 1, '041500002', '415240101', '415', '08:00:00', '17:30:00', '111111', 'CHECKED_OUT'),
+       (current_date - 1, '041500003', '415240101', '415', '09:00:00', '17:30:00', '111111', 'CHECKED_OUT'),
+       (current_date - 1, '041500004', '415240101', '415', '09:00:00', '17:30:00', '111111', 'CHECKED_OUT'),
+       (current_date - 1, '041500005', '415240101', '415', '09:00:00', '17:30:00', '111111', 'CHECKED_OUT'),
```

Comment:
Should include some jurors that should not be added such as EXPENSE_ENTERED,
EXPENSE_EDITED etc to ensure they get filtered out

Reply:
updated